storage/pushsync/pushsync_simulation_test.go

```diff
@@ -191,7 +193,7 @@ func newServiceFunc(ctx *adapters.ServiceContext, bucket *sync.Map) (node.Servic
 
 	bucket.Store(bucketKeyNetStore, netStore)
 
-	noSyncing := &stream.RegistryOptions{Syncing: stream.SyncingDisabled}
+	noSyncing := &stream.RegistryOptions{Syncing: stream.SyncingDisabled, SyncUpdateDelay: 50 * time.Millisecond}
```

Comment:
This is rather confusing to me. We have syncing disabled, but we also need to modify the `SyncUpdateDelay` to 50ms (I know by default it is 10 or 15sec.). Why is that?